__docs__/wonder-blocks-tabs/navigation-tabs.stories.tsx

```diff
@@ -299,7 +299,7 @@ export const HeaderWithNavigationTabsExample: StoryComponentType = {
             navigationTabsRoot: {
                 // set margin to negative value of header vertical spacing so
                 // that selected indicator lines up with header border
-                margin: `-${headerVerticalSpacing} 0`,
+                margin: `calc(${headerVerticalSpacing}, *-1) 0`,
```

Comment:
Is the comma after the token intended? I think this might be why there's an unexpected Chromatic diff for this story!

Reply:
that's why, thanks for noticing that :)